### PR TITLE
Add Icecast source info

### PR DESCRIPTION
**Pull request by meyskens**
Add Icecast source info to the link page with a toggle swithc

---
This change is [Reviewable](https://reviewable.io/reviews/innovate-technologies/control/64)

### CODE REVIEW

**Comment by meyskens** [comment]
schermafbeelding 2016-08-29 om 11 17 05

**Comment by leoetlino** [comment]
IMO, the design is a bit confusing. This is not actually a setting, but it makes it look like it is. I would suggest something similar to the Client/Reseller or currency switcher on shoutca.st:

    Protocol: [ SHOUTcast v1 | Icecast ]

**Comment by meyskens** [comment]
Agreed

**Comment by meyskens** [comment]
pasted image at 2016_08_29 19_34

**Comment by leoetlino** [comment]
Can you align the switcher to the top? (so the top is aligned with the heading text)

File: src/manage/cast/connection-info/connection-info.html [hunk 1]
```diff
@@ -9,10 +9,10 @@
 </div>
 <section class="panel panel-default">
     <div class="panel-body">
-        <h4>Connection Protocol</h4>
         <div class="row">
             <div class="col-sm-8">
-                <span class="text-warning" ng-show="ctrl.protocol == 'Icecast'"><fa name="warning"></fa> The Icecast prototol may not work properly in all enoders.</span>
+                <h4>Connection Protocol</h4>
+                <span class="text-warning" ng-show="ctrl.protocol == 'Icecast'"><fa name="warning"></fa> The Icecast prototol may not work properly in all encoders.</span>
```
Comment (leoetlino) [inline src/manage/cast/connection-info/connection-info.html]: sorry, I missed this last time: `===`

**Comment by leoetlino** [comment]
LGTM, just squash (rebase -i) :)

---

Reviewed 1 of 2 files at r2, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/innovate-technologies/control/64#-:-KQPtVv2EIp2tdtO8Tdq:b-yeuxio)*

**Pull request merged**